app/models/payment_method.rb

```diff
@@ -0,0 +1,3 @@
+class PaymentMethod < ApplicationRecord
+  belongs_to :user
```

Comment:
should we require token to be unique in here? seems like we want to avoid multiple payment methds with the same token getting generated